DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: REDCapTidieR
 Type: Package
 Title: Extract 'REDCap' Databases into Tidy 'Tibble's
-Version: 1.1.0
+Version: 1.1.0.9000
```

Comment:
I think we actually don't want to bump this if we plan to release this as part of `1.1.0`

Reply:
Oh, yea right. 👍